lua/lspconfig/perlpls.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.perlpls = {
+  default_config = {
+    cmd = {"pls"};
+    settings = {
+      perl = {
+        perlcritic = { enabled = false };
+      };
+    };
+    filetypes = {"perl"};
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname) or vim.fn.getcwd()
+    end;
+  };
+  docs = {
+    package_json = "https://raw.githubusercontent.com/FractalBoy/perl-language-server/master/client/package.json";
+    description = [[
+https://github.com/FractalBoy/perl-language-server
```

Comment:
Can you add more description on how to install this? see pyright

Reply:
I'm not the author of PLS so I don't have write access to that github page. I am working with the author to get Neovim instructions included. Part of that process is to get this configuration included in nvim-lspconfig.

Reply:
Oh no worries, we usually add a small snippet here as well